tests/integration/chunkstore/test_chunkstore.py

```diff
@@ -537,7 +537,7 @@ def write_random_data(chunkstore_lib, name, month, days, securities, chunk_size=
     month: integer
     days: list of integers
     securities: list of integers
-    chunk_size: one of 'D', 'M', 'Y'
+    chunk_size: one of 'D', 'M', 'A'
```

Comment:
I changed the code to force 'A' always. Don't want to write data with 'Y' chunksize in Mongo doc, then later read it out with new version of pandas and fail, so now chunksize will be 'A' if user chooses 'Y'